docs/source/howto/data.rst

```diff
@@ -249,11 +247,693 @@ The node class has various methods to set these attributes, such as :py:`~aiida.
+    |            |             |                                          | stored in the attributes or in   |
+    |            |             |                                          | the extras) has a given length   |
+    |            |             |                                          | **(currently implemented only    |
+    |            |             |                                          | in the SQLA backend)**           |
+    +------------+-------------+------------------------------------------+----------------------------------+
+    | shorter    |    lists    | | 'attributes.mylist': {'shorter': 4}    | Check that a list (typically     |
+    |            |             |                                          | stored in the attributes or in   |
+    |            |             |                                          | the extras) has a length shorter |
+    |            |             |                                          | than the specified value         |
+    |            |             |                                          | **(currently implemented only    |
+    |            |             |                                          | in the SQLA backend)**           |
+    +------------+-------------+------------------------------------------+----------------------------------+
+    | longer     |    lists    | | 'attributes.mylist': {'longer': 4}     | Check that a list (typically     |
+    |            |             |                                          | stored in the attributes or in   |
+    |            |             |                                          | the extras) has a length longer  |
+    |            |             |                                          | than the specified value         |
+    |            |             |                                          | **(currently implemented only    |
+    |            |             |                                          | in the SQLA backend)**           |
+    +------------+-------------+------------------------------------------+----------------------------------+
+    | contains   |    lists    | | 'attributes.mykey': {'contains':       | Check that a list (typically     |
+    |            |             |   ['a','b']}                             | stored in the attributes or in   |
+    |            |             |                                          | the extras) contains some        |
+    |            |             |                                          | specific elements or values      |
+    |            |             |                                          | **(currently implemented only    |
+    |            |             |                                          | in the SQLA backend)**           |
+    +------------+-------------+------------------------------------------+----------------------------------+
```

Comment:
For my group meeting presentation, I reformatted these two tables into one and correctly used inline code tags for the operators and examples and fixed the formatting/spacing. I am pasting it here so you can include it:
```

This is a list of all implemented operators:


+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
|**Operator**  |**Datatype** |  **Example**                                          | Explanation                                                                  |
+==============+=============+=======================================================+==============================================================================+
|   ``==``     |      all    | ``'id': {'==': 123}``                                 | Filter for equality                                                          |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
|   ``in``     |      all    | ``'name': {'in': ['foo', 'bar']}``                    | Filter for values that are in the given list.                                |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``>,<,<=,>=``| float,      | ``'ctime': {'<': datetime(2016, 03, 03)}``            | Filter for values that are greater or smaller than a certain value           |
|              | integer,    |                                                       |                                                                              |
|              | date        |                                                       |                                                                              |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``like``     | string      | ``'name': {'like': 'label%'}``                        | Filter for matching substrings where ``%`` and ``_`` are wildcards.          |
|              |             |                                                       | To match a literal ``%`` or ``_`` escape it by prefixing it with ``\\``.     |
|              |             |                                                       |                                                                              |
|              |             |                                                       |                                                                              |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``ilike``    | string      | ``'name': {'ilike': 'lAbEl%'}``                       | Case insensitive version of ``like``.                                        |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``or``       | list of     | ``'id': {'or': [{'<': 12}, {'==': 199}]}``            | A list of expressions where at least one should be matched.                  |
|              | expressions |                                                       |                                                                              |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``and``      | list of     | ``'id': {'and': [{'<': 12}, {'>': 1}]}``              | A list of expressions where all should be matched.                           |
|              | expressions |                                                       |                                                                              |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``has_key``  | dict        | ``'attributes': {'has_key': 'some_key'}``             | Filter for dictionaries that contain a certain key.                          |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``of_type``  |    any      | ``'attributes.some_key': {'of_type': 'bool'}``        | Filter for values of a certain type.                                         |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``of_length``|    lists    | ``'attributes.some_list': {'of_length': 4}``          | Filter for lists of a certain length.                                        |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``shorter``  |    lists    | ``'attributes.some_list': {'shorter': 4}``            | Filter for lists that are shorter than a certain length.                     |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``longer``   |    lists    | ``'attributes.some_list': {'longer': 4}``             | Filter for lists that are longer than a certain length.                      |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
| ``contains`` |    lists    | ``'attributes.some_key': {'contains': ['a', 'b']}``   | Filter for lists that should contain certain values.                         |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+
```

Reply:
Great, thanks!